packages/cfd/src/Assets/svgs/trading-instruments/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import DerivedFX from './ic-appstore-derived-fx.svg';
+import Synthetics from './ic-appstore-synthetics.svg';
+import BasketIndices from './ic-appstore-basket-indices.svg';
+import Stocks from './ic-appstore-stocks.svg';
+import StockIndices from './ic-appstore-stock-indices.svg';
+import Commodities from './ic-appstore-commodities.svg';
+import Forex from './ic-appstore-forex.svg';
```

Comment:
Let's create a separate folder for these icons as compare accounts and change appstore to compare accounts `ic-compare-accounts-stock-indices`.

Reply:
Its already in a separate folder.